opencti-platform/opencti-graphql/src/database/stix-2-1-converter.ts

```diff
@@ -1664,11 +1659,11 @@ const convertToStix = (instance: StoreCommon): S.StixObject => {
   throw UnsupportedError(`No entity converter available for ${type}`);
 };
 
-export const convertStoreToStix = (instance: StoreCommon): S.StixObject => {
+export const convertStoreToStix_2_1 = (instance: StoreCommon): S.StixObject => {
   if (isEmptyField(instance.standard_id) || isEmptyField(instance.entity_type)) {
     throw UnsupportedError('convertInstanceToStix must be used with opencti fully loaded instance');
   }
-  const converted = convertToStix(instance);
+  const converted = convertToStix_2_1(instance);
   const stix = cleanObject(converted);
```

Comment:
As inside converter already validated and clean the object, not sure is needed to do it here

Reply:
I can't find were in the code the clean and validation is done for the `convertToStix_2_1` converter. can you point me in the code were it is done?